sources/mongodb_pipeline.py

```diff
@@ -0,0 +1,109 @@
+from typing import List
+
+import dlt
+from dlt.common import pendulum
+from dlt.pipeline.pipeline import Pipeline
+from dlt.common.pipeline import LoadInfo
+
+try:
+    from .mongodb import mongodb, mongodb_collection  # type: ignore
```

Comment:
please explain the trick in a comment: that we want to make this script to work as part of the tests and standalone so we try two import methods. first being one used for the tests